qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -68,6 +69,7 @@ def __init__(self, *, win_id, private, parent=None):
         self.textChanged.connect(self.update_completion)
         self.textChanged.connect(self.updateGeometry)
         self.textChanged.connect(self._incremental_search)
```

Comment:
Did you verify that all those methods can deal with an empty text?

Reply:
I didn't find any issues.